CHANGELOG.md

```
@@ -43,6 +46,7 @@
 ## History
 
 - [Changelog](#changelog)
+  - [0.1.16](#0116)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)